.github/workflows/extensions.yml

```diff
@@ -0,0 +1,16 @@
+on: [push]
```

Comment:
Has this yml file been tested yet in Github Actions?

Reply:
I'm working on that

Reply:
The yml file has been tested in Github Actions

Reply:
Thanks, did you get a chance testing in LabGraph repo? Reference: https://github.com/facebookresearch/labgraph/actions/workflows/extensions.yml

Reply:
Hello, I did the test on the forked Labgraph Repo
https://github.com/bennaaym/labgraph/actions/workflows/extensions.yml